### PR TITLE
Format config files and enable `configlet fmt` on CI

### PR DESCRIPTION
**Pull request by jiegillet**
This is a new feature of configlet introduced by Erik.
All changes to the config files were done by configlet.
 
New exercises will need to run `configlet fmt --update --yes`.

### CODE REVIEW

**Comment by ceddlyburge** [comment]
Hi Jeremie, the standardisation of the format of the config files seems like a good idea. Is there a way to check that the changes have been applied in CI (for new / modified config.json's)?

**Review by ceddlyburge — APPROVED** [review]

**Comment by jiegillet** [comment]
Yes, absolutely. Copying Erik's message from the maintainer channels on Slack:

> We now have support to enabled configlet fmt to run in your track's configlet workflow. To enable this, you need to create a .github/org-wide-files-config.toml file with the following config section:
> ```
> [configlet]
> fmt = true
> ```
> Once this file is merged (or modified), the org-wide-files syncing mechanism starts (just like for labels) and you'll receive a PR to update the configlet workflow.
> Here is an example of enabling formatting for the F# track:
> https://github.com/exercism/fsharp/pull/1063#event-6283177219: format all documents (using configlet fmt --update --yes)
> https://github.com/exercism/fsharp/pull/1066: enable configlet ftm by adding the above-mentioned org-wide-files config file
> https://github.com/exercism/fsharp/pull/1067/files: automatic PR to update the workflow

I have added the `.github/org-wide-files-config.toml` file in this PR, so once it is merged, we will receive an automated PR which will update the CI.

**Pull request merged**
